src/janus-api/models/feed.ts

```diff
@@ -224,6 +224,19 @@ export class Feed {
       this.speakObserver.destroy();
     }
     this.connection = null;
+    this.removeStream();
+  };
+
+  /**
+   * Unsets the stream from the feed, stopping all its tracks.
+   */
+  private removeStream() {
+    if (this.stream) {
+      let tracks = this.stream.getTracks();
+      for(let i in tracks) {
```

Comment:
Maybe this also works:

~~~
tracks.map(t => t.stop());
~~~

Reply:
I would use `forEach` because you are not "mapping" at all.

Reply:
ah, right. I  only wanted to point that we can "functionally" iterate.